tests/components/zwave_mqtt/test_switch.py

```diff
@@ -0,0 +1,41 @@
+"""Test Z-Wave Switches."""
+from .common import setup_zwave
+
+
+async def test_switch(hass, generic_data, sent_messages, switch_msg):
+    """Test setting up config entry."""
+    receive_message = await setup_zwave(hass, fixture=generic_data)
+
+    # Test loaded
+    state = hass.states.get("switch.smart_plug_switch")
+    assert state is not None
+    assert state.state == "off"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5460460)